package.json

```diff
@@ -9,56 +9,56 @@
   },
   "dependencies": {
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
-    "browser-sync": "^2.8.0",
-    "capital-framework": "^3.1.1",
-    "del": "^2.0.0",
-    "gulp": "^3.9.0",
-    "gulp-autoprefixer": "^3.1.0",
-    "gulp-changed": "^1.2.1",
-    "gulp-cssmin": "^0.1.7",
-    "gulp-header": "^1.2.2",
-    "gulp-imagemin": "^2.3.0",
-    "gulp-less": "^3.0.3",
-    "gulp-load-plugins": "^1.2.0",
-    "gulp-modernizr": "^1.0.0-alpha",
+    "browser-sync": "2.11.2",
+    "capital-framework": "3.2.1",
```

Comment:
Oh, good point. I was pinning before updating to the release @contolini prepped so that we could see how that would go. Maybe we do that and then unpin it again afterwards?